evmjit/libevmjit/Common.h

```diff
@@ -1,11 +1,12 @@
 #pragma once
 
-#include <vector>
 #include <tuple>
 #include <cstdint>
 
 #ifdef _MSC_VER
-#define EXPORT __declspec(dllexport)
+#define EXPORT __declspec(dllexport)
+#define _ALLOW_KEYWORD_MACROS
+#define noexcept throw()
```

Comment:
Why do you do that? Such defines in headers are dangerous. This could affect more files than intended. Since we are using C++11 we should use noexcept rather then throw()